.github/workflows/test-and-check.yml

```diff
@@ -53,6 +54,42 @@ jobs:
       - name: Check the changesets
         run: node -r esbuild-register tools/deployments/validate-changesets.ts
 
+  check:
+    if: github.ref != 'refs/heads/changeset-release/main'
```

Comment:
This will need to run on all PRs since it is now a required check.

Reply:
Ah—I'd been hoping that the name based matching would make this work. I'll fix that